tests/gflownet/envs/test_composition.py

```diff
@@ -38,15 +38,39 @@ def test__environment__initializes_properly(elements):
     [
         (
             [0, 0, 2, 0],
-            [0, 0, 2, 0],
+            [
+                # fmt: off
+                0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                # fmt: on
```

Comment:
What are these `fmt: off` / `fmt: on` comments?

Reply:
It's a way to tell black not to format these lines (and not to complain about their format) because otherwise black will unroll these lists over 95 lines with 1 element per line.

Reply:
This is very useful to know!